### PR TITLE
Update make_break_var macro to add children break

### PR DESCRIPTION
**Pull request by chedman**

### CODE REVIEW

File: Macros/Make_break_vars_2017.sas [hunk 1]
```diff
@@ -74,6 +74,12 @@
   IF (PPRENT=2) or (PPRENT=3)  THEN homeown = 0;
   IF (PPRENT=1) THEN homeown = 1;
   format homeown homeown.;
+
+  children=.;
+  IF (PPT01=0) or (PPT25=0) or (PPT612=0) or (PPT1317=0) THEN children = 0;
+  IF (PPT01>=1) or (PPT25>=1) or (PPT612>=1) or (PPT1317>=1) THEN children = 1;
+  format children children.;
```
Comment (lhendey) [inline Macros/Make_break_vars_2017.sas:81]: @chedman - did you add a child format to the make formats.sas program?

**Review by lhendey — DISMISSED** [review]
@chedman see my comment - did you add a child format?

**Comment by lhendey** [comment]
@chedman - updated branch with master. you should be able to edit the q15 and 16 recode programs now

**Comment by chedman** [comment]
@lhendey Ok great - I'll look at those now.

**Comment by chedman** [comment]
@lhendey just synced the changes

**Pull request merged**